CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 * Add MDC to the `LoggingMdcFilter` to include API method, path, and request ID.
 
+### Changed
+
+* Upgraded from JDK 11 to JDK 17 [@ucg8j](https://github.com/ucg8j)
+* Switched JDK image from Alpine to regular image to enable Marquez to run on multiple CPU architectures [@ucg8j](https://github.com/ucg8j)
+
```

Comment:
Mind also adding a "Fixed" section for https://github.com/MarquezProject/marquez/issues/1818?